content/influxdb/v2.1/notebooks/clean-data.md

```diff
@@ -10,31 +10,90 @@ menu:
-6. Run your notebook:
-  - Click **Run** to run the notebook and write to the output bucket a single time.
-  - To write continuously, click **Export as Task** in the upper right corner of the **Output to Bucket** cell. For details about working with tasks, see [Manage tasks](/influxdb/v2.1/process-data/manage-tasks/).
+1.  [Create a new notebook](/influxdb/v2.1/notebooks/create-notebook/).
+2.  In the **Build a Query** cell:
+
+    1.  In the **FROM** column under **{{% caps %}}Sample{{% /caps %%}},
+        select **NOAA National Buoy Data**.
+    2.  In the next **FILTER** column, select **_measurement** from the dropdown
+        and select the **ndbc** measurement in the list of measurements.
+    3.  In the next **FILTER** column, select **_field** from the dropdown
+        and select the **wind\_dir\_degt** field from the list of fields.
+
+3.  Click {{% icon "notebook-add-cell" %}} after your **Build a Query** cell to 
+    add a new cell and select **{{% caps %}}Flux Script{{% /caps %}}**.
+    
+4.  In the Flux script cell:
+    
+    1.  Define a custom function (`cardinalDir()`) that converts a numeric degree
+        value to a cardinal direction (N, NNE, NE, etc.).
+    2.  Use `__PREVIOUS_RESULT__` to load the output of the previous notebook
+        cell into the Flux script.
+    3.  Use [`map()`](/{{< latest "flux" >}}/stdlib/universe/map/) to iterate
+        over each input row, update the field key to `wind_dir_cardinal`, and
+        normalize the `_value` column to a cardinal direction using the custom
+        `cardinalDir()` function.
+    
+    ```js
+    import "array"
+
+    cardinalDir = (d) => {
+        _cardinal = if d >= 348.7 or d < 11.25 then "N"
+        else if d >= 11.25 and d < 33.75 then "NNE"
+        else if d >= 33.75 and d < 56.25 then "NE"
+        else if d >= 56.25 and d < 78.75 then "ENE"
+        else if d >= 78.75 and d < 101.25 then "E"
+        else if d >= 101.25 and d < 123.75 then "ESE"
+        else if d >= 123.75 and d < 146.25 then "SE"
+        else if d >= 146.25 and d < 168.75 then "SSE"
+        else if d >= 168.75 and d < 191.25 then "S"
+        else if d >= 191.25 and d < 213.75 then "SSW"
+        else if d >= 213.75 and d < 236.25 then "SW"
+        else if d >= 236.25 and d < 258.75 then "WSW"
+        else if d >= 258.75 and d < 281.25 then "W"
+        else if d >= 281.25 and d < 303.75 then "WNW"
+        else if d >= 303.75 and d < 326.25 then "NW"
+        else if d >= 326.25 and d < 348.75 then "NNW"
+        else ""
+
+        return _cardinal
+    }
+
+    __PREVIOUS_RESULT__
+        |> map(fn: (r) => ({r with
+            _field: "wind_dir_cardinal",
+            _value: cardinalDir(d: r._value),
+        }))
+    ```
+
+5.  Click {{% icon "notebook-add-cell" %}} after your **Flux Script** cell to 
+    add a new cell and select **{{% caps %}}Output to Bucket{{% /caps %}}**.
```

Comment:
Output to bucket is no longer an option--how to add output to Flux script

Reply:
Ok, another difference between oss and cloud. I'll fix it.